assembly/src/assembler/context.rs

```diff
@@ -335,8 +335,8 @@ impl ModuleContext {
 
     /// Returns a [Procedure] with the specified ID, or None if a compiled procedure with such ID
```

Comment:
Would be good to update the comment to reflect the fact the lookup is now done against the mast root.